modules/glfw/canvasglfw.cpp

```diff
@@ -186,6 +186,7 @@ void CanvasGLFW::character(GLFWwindow* window, unsigned int character) {
     // Needed for text input
     auto thisCanvas = getCanvasGLFW(window);
     // Convert UTF32 character
+
```

Comment:
This should probably not be in the PR...

Reply:
This one was fixed on my machine and on master so it was resulting in a conflict. I merged them, this is why the commit is on the branch.

Reply:
Generally we never merge master into a feature branch. You would instead rebase the feature branch on top of the latest master.